safehttp/plugins/csp/internalunsafecsp/internalunsafecsp.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package internalunsafecsp is used internally to override CSP.
```

Comment:
Should this be in an `internal` subdirectory if it is only for internal use? The Go tool will not let you import an `internal` package from another module.

Reply:
No, other packages could have to import this or refer to this. This "internal" will only be enforced with allowlists with either bancheck or visiblity restrictions.

Same for the others.